### PR TITLE
New changes#Homework17

### PR DESCRIPTION
**Pull request by ArturH1990**
changes#Homework17

### CODE REVIEW

**Comment by duykhanhrc** [comment]
// comment from hw team: Hi @ArturH1990 , Test script is failed! You got this problem, please correct it.

image

File: src/test/java/Homework17.java [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+
+        String url = "https://qa.koel.app/";
+        driver.get(url);
+
+        WebElement emailInput = driver.findElement(By.cssSelector("input[type='email']"));
+        emailInput.click();
+        emailInput.clear();
+        emailInput.sendKeys("art1234@mail.com");
+
+        WebElement passwordInput = driver.findElement(By.cssSelector("Password"));
```
Comment (duykhanhrc) [inline src/test/java/Homework17.java:30]: I see you get password input selector at here wrong, please check and get correct.
// HINT: `WebElement passwordInput = driver.findElement(By.cssSelector("input[type=password]"));`

File: src/test/java/Homework17.java [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+
+            WebElement choosePlaylist = driver.findElement(By.cssSelector("section[id='songResultsWrapper'] li[class='favorites']"));
+            choosePlaylist.click();
+
+
+            WebElement messageT = driver.findElement(By.cssSelector("section[id='songResultsWrapper'] li[class='favorites']"));
+            Assert.assertEquals(messageT.getText(), "Added 1 song into {favorites}");
+
+
+            driver.quit();
```
Comment (duykhanhrc) [inline src/test/java/Homework17.java:60]: You can add `driver.quit();` to `@BeforeTest` or `@BeforeMethod`. It quit browser after test or method and, will avoid the case if your test script fails but haven't gotten to this part yet -> will not quit browser

File: src/test/java/Homework17.java (hunk 1 above)
Comment (ArturH1990) [reply src/test/java/Homework17.java:30]: thanks, I corrected it

File: src/test/java/Homework17.java (hunk 2 above)
Comment (ArturH1990) [reply src/test/java/Homework17.java:60]: ok, but I am getting this error message...

Screenshot 2023-06-26 at 22 34 43

Comment (ArturH1990) [reply src/test/java/Homework17.java:60]: again

**Pull request merged**